tests/system/src/test/java/test/javafx/scene/lighting3D/PointLightAttenuationTest.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package test.javafx.scene.lighting3D;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertEquals;
+
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import javafx.application.Application;
+import javafx.application.Platform;
+import javafx.scene.Group;
+import javafx.scene.PointLight;
+import javafx.scene.Scene;
+import javafx.scene.paint.Color;
+import javafx.scene.shape.Box;
+import javafx.stage.Stage;
+import javafx.stage.WindowEvent;
+import test.util.Util;
+
+public class PointLightAttenuationTest {
+
+    private static final double DELTA = 1d/255; // smallest color resolution
+    private static final int LIGTH_DIST = 60;
+    private static final int SAMPLE_DIST = 60;
+
+    private static CountDownLatch startupLatch;
+    private static Stage stage;
+    private static PointLight light = new PointLight(Color.BLUE);
+    private static Box box = new Box(150, 150, 1);
+
+    public static void main(String[] args) throws Exception {
+        initFX();
+    }
+
+    @BeforeClass
+    public static void initFX() throws Exception {
```

Comment:
One more thing I forgot to mention. You will need to ship the test if 3D is not supported using `assumeTrue`. See [TriangleMeshValidationTest.java#L64](https://github.com/openjdk/jfx/blob/master/tests/system/src/test/java/test/robot/test3d/TriangleMeshValidationTest.java#L64).